op-node/rollup/interop/managed/system.go

```diff
@@ -275,3 +276,29 @@ func (m *ManagedMode) BlockRefByNumber(ctx context.Context, num uint64) (eth.Blo
 func (m *ManagedMode) ChainID(ctx context.Context) (supervisortypes.ChainID, error) {
 	return supervisortypes.ChainIDFromBig(m.cfg.L2ChainID), nil
 }
+
+func (m *ManagedMode) OutputV0AtTimestamp(ctx context.Context, timestamp uint64) (*eth.OutputV0, error) {
+	num, err := m.cfg.TargetBlockNumber(timestamp)
+	if err != nil {
+		return nil, err
+	}
+	ref, err := m.l2.L2BlockRefByNumber(ctx, num)
+	if err != nil {
+		return nil, err
+	}
+	return m.l2.OutputV0AtBlock(ctx, ref.Hash)
+}
+
+func (m *ManagedMode) PendingOutputV0AtTimestamp(ctx context.Context, timestamp uint64) (*eth.OutputV0, error) {
+	num, err := m.cfg.TargetBlockNumber(timestamp)
+	if err != nil {
+		return nil, err
+	}
+	ref, err := m.l2.L2BlockRefByNumber(ctx, num)
+	if err != nil {
+		return nil, err
+	}
+	_ = ref
+	// TODO: return pending output in the optimistic block deposited transaction - https://github.com/ethereum-optimism/specs/pull/489
+	panic("not implemented")
```

Comment:
Wondering if we should just always return the canonical block for now since re-orgs aren't yet supported.  Not sure we should merge with a panic here as it would take out the whole process if an RPC request is received which doesn't seem ideal.

Or just hard code 0x00000 so it's obviously wrong...

Reply:
Yeah, good idea. I'll go with the canonical output so we can at least test the happy path e2e.